src/FSharpSource.Settings.targets

```diff
@@ -85,12 +85,12 @@
     <!-- Frozen FSharp.Core package being built with this build -->
     <FSharpCore41TargetPackageVersion>4.1.19</FSharpCore41TargetPackageVersion>
     <FSharpCore41TargetMajorVersion>4.1</FSharpCore41TargetMajorVersion>
-    <FSharpCoreLatestTargetPackageVersion>4.3.0</FSharpCoreLatestTargetPackageVersion>
+    <FSharpCoreLatestTargetPackageVersion>4.3.1</FSharpCoreLatestTargetPackageVersion>
     <FSharpCoreLatestTargetMajorVersion>4.3</FSharpCoreLatestTargetMajorVersion>
 
     <!-- Frozen FSharp.Core package -->
     <FSharpCoreFrozenPortablePackageVersion>10.1.0</FSharpCoreFrozenPortablePackageVersion>
-    <FSharpCoreFrozenPortableTargetPackageVersion>10.1.1</FSharpCoreFrozenPortableTargetPackageVersion>
+    <FSharpCoreFrozenPortableTargetPackageVersion>10.2.0</FSharpCoreFrozenPortableTargetPackageVersion>
```

Comment:
I thought this was supposed to be frozen.

Reply:
We update the version number to match the next point at which it might get re-frozen.  I should have updated it last time, but forgot.  Effectively we were rebuild the version we had already shipped.